component/vshn_codey.jsonnet

```diff
@@ -115,6 +116,24 @@ local composition =
               ],
             },
           },
+          {
+            step: 'codey-func',
```

Comment:
I decided to just make codey work with ourrelease process even though it will probably fail due to maintenance nor running properly

Reply:
We might want to add some logic to the appcat part though, because we should run the maintenance job for codey at the same time as the underlying vshnforgejo is running. Or a few minutes apart at most.

But that can be handled in another story.

EDIT: Also the underlying maintenance for vshnforgejo is working.